PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,13 +2,14 @@
 
 Please include a summary of the change with relevant motivation and context
 
-## Linked issues
+## Linked Issues
 
 - (issue)
```

Comment:
should we just get rid of this section then?

Reply:
Keep it for now because we can't link issues from other repositories